distributed/scheduler.py

```diff
@@ -2978,40 +2784,52 @@ def start_ipython(self, stream=None):
             )
         return self._ipython_kernel.get_connection_info()
 
+    def worker_objective(self, key, worker):
+        """
+        Objective function to determine which worker should get the key
+
+        Minimize expected start time.  If a tie then break with data storate.
+        """
+        comm_bytes = sum([self.nbytes.get(k, 1000) for k in self.dependencies[key]
```

Comment:
I think this is not the first time the `1000` default value pops up, perhaps a constant would make things easier?